src/NLog/Layouts/JSON/JsonLayout.cs

```diff
@@ -341,13 +347,18 @@ private void BeginJsonProperty(StringBuilder sb, string propName, bool beginJson
         {
             if (beginJsonMessage)
             {
-                sb.Append(SuppressSpaces ? "{\"" : "{ \"");
+                sb.Append((IndentJson || SuppressSpaces) ? "{" : "{ ");
             }
             else
             {
-                sb.Append(SuppressSpaces ? ",\"" : ", \"");
+                sb.Append((IndentJson || SuppressSpaces) ? "," : ", ");
             }
 
+            if (IndentJson)
+                sb.AppendLine().Append("\t");
```

Comment:
Make this a single character `sb.AppendLine().Append('\t');`, instead of string.